validation/chOpenApiValidator.go

```diff
@@ -98,20 +98,20 @@ func (chv CHValidatorImpl) ValidateRequestAgainstOpenApiSpec(httpReq *http.Reque
 		// Switch off the addition of schema error details to the returned error. This stops the OpenApi schema being added to errors.
 		openapi3.SchemaErrorDetailsDisabled = true
 
-		log.Info("Validating request...", nil)
+		log.InfoC(contextId, "Validating request...", nil)
 		if err := callOpenApiFilterValidateRequest(ctx, requestValidationInput); err != nil {
 			// If errors are found in the request format them and return them.
-			log.Info("Request validated. Errors found.", nil)
-			return callFormatError(err), nil
+			log.InfoC(contextId, "Request validated. Errors found.", nil)
```

Comment:
update to use log data please

Reply:
Data included

Reply:
The message mentions "Request validated. Errors found" should we still be logging nil in data ?

Reply:
Changed it to ErrorC